components/_host-profiles.scss

```diff
@@ -0,0 +1,63 @@
+/*
+
+Stylesheet for all things related to presentation of 
+host profiles.
+
+*/
+
+.all-bio-content {
+	overflow: auto;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered margin-bottom, overflow